### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by AryanK1511**
This solves #11

### CODE REVIEW

**Review by KaosElegent — APPROVED** [review]
Looks good, but you can do better!

File: content.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is the content to put on the website
```
Comment (KaosElegent) [inline content.md:1]: Yoo

Comment (AryanK1511) [reply content.md:1]: Sounds good lol

Comment (KaosElegent) [inline content.md:1]: dsgsdfg

**Review by KaosElegent — APPROVED** [review]
sdfgsdf

File: content.md (hunk 1 above)
Comment (AryanK1511) [reply content.md:1]: bro chill

**Review by AbdulmuhsinB — APPROVED** [review]
Nice

**Comment by gurmehakkaur** [comment]
reviewed

**Review by AbdulmuhsinB — APPROVED** [review]
Approved

**Review by AbdulmuhsinB — APPROVED** [review]
Approving

**Review by kanwar1413 — COMMENTED** [review]
✅

**Review by Harshp258 — COMMENTED** [review]
Works

**Comment by joaovitortc** [comment]
Wow!

**Pull request merged**